src/vs/workbench/contrib/chat/browser/chatEditing/chatEditingSession.ts

```diff
@@ -999,6 +1005,13 @@ export class ChatEditingSession extends Disposable implements IChatEditingSessio
 		entry.acceptAgentEdits(textEdits, isLastEdits, responseModel);
 	}
 
+	private async _acceptNotebookEdits(resource: URI, edits: ICellEditOperation[], isLastEdits: boolean, responseModel: IChatResponseModel): Promise<void> {
+		const entry = await this._getOrCreateModifiedFileEntry(resource, this._getTelemetryInfoForModel(responseModel)) as IModifiedFileEntry;
```

Comment:
Casting into `IModifiedFileEntry` for now as I think this is debt that you're working on.